pkg/proto/types.go

```diff
@@ -4221,12 +4221,42 @@ func (s *StateHash) toStateHashJS() stateHashJS {
 	}
 }
 
-type StateHashJSDebug struct {
+type StateHashDebug struct {
 	stateHashJS
 	Height  uint64 `json:"height"`
```

Comment:
Consider adding `omitempty` to height and version fields. What if we request the previous version of node that doesn't support new extended state hash API?